libs/java/client_common/src/main/java/com/yahoo/athenz/common/utils/X509CertUtils.java

```diff
@@ -223,16 +223,7 @@ public static String extractIssuerDn(X509Certificate[] certs) {
             return "";
         }
 
-        return extractIssuerDn(certs[0]);
-    }
-
-    /**
-     * extractIssuerDn returns the IssuerDN from the certificate passed in
-     * @param cert X509Certificate to extract the DN from
-     * @return string with Issuer DN
-     */
-    public static String extractIssuerDn(X509Certificate cert) {
```

Comment:
we cannot move public methods from common client libraries since it will break any users of this method. So we can change the implementation to call the method fro Crypto class but not remove it.